### DISCUSSION TITLE
playMusicStream will not play if the application is on a different vitrual space

### DISCUSSION THREAD

**Discussion by delyan-kirov** [Anut-py/h-raylib#53] [General]
Basically title. I don't think this is the same behavior with the native `raylib` version, but I'm not sure. The function `playSound` will play in the background. I just wanted to know if this is the intended behavior and if it is, is there a way to make it play in the background? The reason why I wanted to use this function is because `playSound` doesn't have a nice way of seeking to a point in the track.

**Comment by Anut-py** [comment]
This sounds like something with raylib itself, not h-raylib. You should probably ask in the raylib github page or discord server.

**Reply by delyan-kirov** [reply]
The problem is actually not `playMusicStream`, it's that `playMusicStream` requires generating new frames while `playSound` does not. The application simply does not continue rendering if it's on a different work space, so no audio plays. I'll have to look how to make rendering continue even if the window is not in the current work space.
